client-participation/polis.config.template.js

```diff
@@ -6,13 +6,14 @@ module.exports = {
     "^192\\.168\\.1\\.140$",
     "^pol\\.is",
     ".+\\.pol\\.is$",
+    ".+\\.polisdemo\\.tech$",
     "^xip\\.io$",
     ".+\\.xip\\.io$",
   ],
 
   // Point to a polisServer instance (local recommended for dev)
   //SERVICE_URL: "http://localhost:5000", // local server; recommended for dev
-  SERVICE_URL: "https://preprod.pol.is",
+  SERVICE_URL: "http:localhost:5000",
```

Comment:
Did you find this to affect anything?

Reply:
No, but I feel that the defalut "dev" setup should not interact with https://preprod.pol.is or https://pol.is, but should use localhost. When we need to use Docker to build and deploy to production, the appropriate ENV vars should override the local stuff with the production stuff.

Reply:
Agreed, having preprod.pol.is seemed odd / confusing when I was setting this up

Reply:
Heh yeah, I just never teased apart what all these references did, so was wondering!